compile/transform.ts

```diff
@@ -140,7 +140,7 @@ export const transformMouldToReactComponent = (mould: Mould): string => {
             )`
     }
 
-    return `export const ${mouldName} = (props) => {
+    return `export function ${mouldName}(props) {
```

Comment:
Because of hoisting, using function declare components could do not care of order